Content.Shared/Storage/Components/SharedEntityStorageComponent.cs

```diff
@@ -113,6 +113,12 @@ public abstract partial class SharedEntityStorageComponent : Component
     [DataField]
     public EntityWhitelist? Whitelist;
 
+    /// <summary>
+    ///     Blacklist for what entities are not allowed to be inserted into this container.
+    /// </summary>
+    [DataField]
+    public EntityWhitelist? Blacklist;
+
```

Comment:
Mixing blacklists and whitelists together makes managing it a nightmare, better to just stick to a whitelist. This problem isn't going to be exclusive to mobs either.